src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -865,8 +874,44 @@ private JButton getCloseButton() {
 		});
 		return closeButton;
 	}
-	
-	private JButton getCleanTableButton() {
+
+	private JButton getPrintButton() {
+		printButton = new JButton(MessageBundle.getMessage("angal.common.print.btn"));
+		printButton.setMnemonic(MessageBundle.getMnemonic("angal.common.print.btn.key"));
+		printButton.setEnabled(true);
+
+		printButton.addActionListener(e -> {
+			if (inventory == null || inventory.getId() <= 0) {
```

Comment:
This is more type safe because getId() might be null and unboxing could return a NPE

```suggestion
			if (inventory == null || inventory.getId() == null || inventory.getId() <= 0) {
```